presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsTracker.java

```diff
@@ -59,6 +60,8 @@
 {
     private static final Logger LOG = Logger.get(HistoryBasedPlanStatisticsTracker.class);
     private static final Set<QueryType> ALLOWED_QUERY_TYPES = ImmutableSet.of(SELECT, INSERT);
+    // Variable name prefix for hashes created by HashGenerationOptimizer
+    private static final String HASH_VALUE_PREFIX = "$hashValue";
```

Comment:
could you point out where exactly this variable is generated?

Reply:
HashGenerationOptimizer

Reply:
It comes from https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/sql/planner/PlanVariableAllocator.java#L83.  Would be good to extract a static variable there to share between the two.